app/models/doi.rb

```diff
@@ -878,6 +878,15 @@ def self.delete_test_dois(from_date: nil)
     end
   end
 
+  #to be used after DOIS transfer to another RA
+  def self.delete_dois_by_prefix(prefix)
```

Comment:
Will the destroy trigger a removal from the ES index?

Reply:
indexable is included in the DOI model, and it adds a trigger for before destroy 

https://github.com/datacite/lupo/blob/d4b26a9e41e29c926211e54c908b8ba9d6b76342/app/models/concerns/indexable.rb#L16